replica/table.cc

```diff
@@ -2250,19 +2266,16 @@ table::make_reader_v2_excluding_sstables(schema_ptr s,
 
 future<> table::move_sstables_from_staging(std::vector<sstables::shared_sstable> sstables) {
```

Comment:
function should be renamed to remove_sstables_from_.. instead

Reply:
done